test/unit/range/view/view_pairwise_combine_test.cpp

```diff
@@ -0,0 +1,270 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <list>
+#include <utility>
+#include <vector>
+
+#include <range/v3/algorithm/equal.hpp>
+#include <range/v3/view/filter.hpp>
+
+#include <seqan3/range/view/pairwise_combine.hpp>
+#include <seqan3/test/pretty_printing.hpp>
+
+template <typename t>
+class pairwise_combine_base_test : public ::testing::Test
+{
+public:
+
+    using view_t       = decltype(seqan3::detail::pairwise_combine_view{std::declval<t &>()});
+    using const_view_t = decltype(seqan3::detail::pairwise_combine_view{std::declval<t const &>()});
+
+    auto create_view()
+    {
+        return view_t{container | seqan3::view::all};
```

Comment:
I thought the constructor will wrap a container in view all itself. You should be able to construct from any container then without the pipe

Reply:
good point 😅